common/lib/dependabot/file_updaters/vendor_updater.rb

```diff
@@ -18,7 +18,9 @@ def updated_vendor_cache_files(base_directory:)
         return [] unless repo_contents_path && vendor_dir
 
         Dir.chdir(repo_contents_path) do
-          relative_dir = Pathname.new(vendor_dir).relative_path_from(Dir.pwd)
+          relative_dir = Pathname.new(vendor_dir).relative_path_from(
+            repo_contents_path
+          )
```

Comment:
Not sure how to test this 🤔

Reply:
Only seems to happen on mac, current tests are passing so 👍 I guess